streamer/src/nonblocking/quic.rs

```diff
@@ -95,7 +95,7 @@ pub fn spawn_server(
 
 pub async fn run_server(
     mut incoming: Incoming,
-    packet_sender: Sender<PacketBatch>,
+    packet_sender: Sender<Batch<TransactionPacket>>,
```

Comment:
Can we make this generic as well? the streamer does not necessarily receive only transaction packets. We are working to use quic to support repair/gossip as well.

Reply:
Yeah definitely. I didn't do it here to try to reduce the complexity of the PR, but I can add an issue to do it as a follow-up if it's ok with you.

Reply:
It ended up being easier to do it during a rebase, so this is done now!